sdk/python/jobs/automl-standalone-jobs/automl-distributed-regression-task-hardware-performance/regression-distributed-lightgbm.ipynb

```diff
@@ -0,0 +1,1505 @@
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": false,
+     "outputs_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    },
+    "gather": {
+     "logged": 1684768766205
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "###### Show Azure ML Workspace information"
+   ],
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "workspace = ml_client.workspaces.get(name=ml_client.workspace_name)\n",
+    "\n",
+    "output = {}\n",
+    "output[\"Workspace\"] = ml_client.workspace_name\n",
+    "output[\"Subscription ID\"] = ml_client.connections._subscription_id\n",
+    "output[\"Resource Group\"] = workspace.resource_group\n",
+    "output[\"Location\"] = workspace.location\n",
+    "output"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": false,
+     "outputs_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    },
+    "gather": {
+     "logged": 1684768773911
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## 2. Data\r\n",
+    "\r\n",
+    "We will download the data and save it in the respective folders, created at the same location (in the repo) as this notebook.\r\n",
+    "For large data we recommend that the dataset be registered in your workspace."
+   ],
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "import urllib\n",
+    "from zipfile import ZipFile\n",
+    "\n",
+    "data_path = \"./data/\"\n",
+    "os.makedirs(data_path, exist_ok=True)\n",
+    "\n",
+    "# Download dataset files and copy them in data path\n",
+    "\n",
+    "data_download_url = \"https://automlsamplenotebookdata.blob.core.windows.net/automl-sample-notebook-data/machineData.csv\"\n",
+    "data_file = data_path + \"data.csv\"\n",
+    "urllib.request.urlretrieve(data_download_url, filename=data_file)\n",
+    "\n",
+    "print(\"Dataset files downloaded...\")"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": false,
+     "outputs_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    },
+    "gather": {
+     "logged": 1684768783335
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "#### Create the Azure Machine Learning MLTable"
+   ],
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "def create_folder_and_ml_table(csv_file, output, delimiter=\",\", encoding=\"ascii\"):\n",
```

Comment:
There is now an mltable API that you can use to replace this code. See: [mltable](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-mltable?view=azureml-api-2&tabs=cli). My code samples in the upcoming public docs changes also use the mltable APIs. Here's a sampe:
``
import mltable

paths = [
    {'file': './train_data/timeseries_train.csv'}
]

train_table = mltable.from_delimited_files(paths)
train_table.save('./train_data')
``

Reply:
Actually, I think we need to hold off on adding an mltable API example until the newest runtime CE ships. In my own experiment, I ran into an error that has been since been fixed. The github issue for it is [here](https://github.com/Azure/azure-sdk-for-python/issues/29480)

Reply:
Got it. So, leaving it as it was originally.